Dialog/core/designsystem/src/commonMain/kotlin/com/on/dialog/designsystem/component/button/Button.kt

```diff
@@ -0,0 +1,202 @@
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.OutlinedButton
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.on.dialog.designsystem.icon.DialogIcons
+import com.on.dialog.designsystem.theme.DialogTheme
+import org.jetbrains.compose.ui.tooling.preview.Preview
+
+@Composable
+fun DialogButton(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
+    contentPadding: PaddingValues = ButtonDefaults.ContentPadding,
+    shape: Shape = RoundedCornerShape(DialogButtonDefaults.Default.cornerRadius),
+    content: @Composable RowScope.() -> Unit,
+) {
+    Button(
+        onClick = onClick,
+        modifier = modifier,
+        enabled = enabled,
+        colors =
+            ButtonDefaults.buttonColors(
+                containerColor = MaterialTheme.colorScheme.primaryContainer,
+            ),
+        contentPadding = contentPadding,
+        content = content,
+        shape = shape,
+    )
+}
+
+@Composable
+fun DialogButton(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
+    text: @Composable () -> Unit,
+    leadingIcon: @Composable (() -> Unit)? = null,
+) {
+    DialogButton(
+        onClick = onClick,
+        modifier = modifier,
+        enabled = enabled,
+        contentPadding =
+            if (leadingIcon != null) {
+                ButtonDefaults.ButtonWithIconContentPadding
+            } else {
+                ButtonDefaults.ContentPadding
+            },
+    ) {
+        DialogButtonContext(
+            text = text,
+            leadingIcon = leadingIcon,
+        )
+    }
+}
+
+@Composable
+fun DialogOutlinedButton(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
+    contentPadding: PaddingValues = ButtonDefaults.ContentPadding,
+    shape: Shape = RoundedCornerShape(DialogButtonDefaults.Default.cornerRadius),
+    content: @Composable RowScope.() -> Unit,
+) {
+    OutlinedButton(
+        onClick = onClick,
+        modifier = modifier,
+        enabled = enabled,
+        colors =
+            ButtonDefaults.outlinedButtonColors(
+                contentColor = MaterialTheme.colorScheme.onBackground,
+            ),
+        border =
+            BorderStroke(
+                width = DialogButtonDefaults.Default.outlinedButtonBorderWidth,
+                color =
+                    if (enabled) {
+                        MaterialTheme.colorScheme.outline
+                    } else {
+                        MaterialTheme.colorScheme.onSurface.copy(
+                            alpha = DialogButtonDefaults.Default.disabledOutlinedButtonBorderAlpha,
+                        )
+                    },
+            ),
```

Comment:
```suggestion
border =
        DialogButtonDefaults.outlinedButtonBorder(enabled)
```
object로 바꾼 default면 다음과 같은 방식으로 변경될 수 있을 거 같습니다.😊